Sources/RoktUXHelper/Utils/Optional+Extension.swift

```diff
@@ -12,6 +12,16 @@
 import Foundation
 
 extension Optional {
+
+    var isEmpty: Bool {
+        switch self {
+        case .none:
+            true
+        case .some:
+            false
+        }
+    }
```

Comment:
Suggestion for my other comment

```suggestion
    var isEmpty: Bool {
        switch self {
        case .none:
            true
        case .some:
            false
        }
    }
    
    var isNotEmpty: Bool {
        !isEmpty
    }
```